.cirrus.yml

```diff
@@ -85,6 +87,7 @@ task:
         cpu: 4
         memory: 12G
     - name: tests_extras-linux
+      skip: "!changesInclude('packages/flutter/**', 'packages/flutter_test/**', 'packages/flutter_tools/lib/src/test/**', 'bin/internal/**') && $CIRRUS_BRANCH != 'master'"
```

Comment:
We should probably include `dev/**` in these.

Reply:
Done